system_metrics_collector/src/system_metrics_collector/linux_cpu_measurement_node.hpp

```diff
@@ -85,6 +85,18 @@ class LinuxCpuMeasurementNode : public system_metrics_collector::PeriodicMeasure
    */
   virtual system_metrics_collector::ProcCpuData makeSingleMeasurement();
 
+  /**
+   * This publishes the statistics derived from the collected measurements
+   */
+  void publishStatistics() override;
```

Comment:
See comments regarding the interface further down.

Reply:
Done.